modules/service/src/main/scala/lucuma/odb/graphql/mapping/ObsAttachmentMapping.scala

```diff
@@ -22,7 +40,34 @@ trait ObsAttachmentMapping[F[_]] extends ObsAttachmentTable[F] with ProgramTable
         SqlField("checked", ObsAttachmentTable.Checked),
         SqlField("fileSize", ObsAttachmentTable.FileSize),
         SqlField("updatedAt", ObsAttachmentTable.UpdatedAt),
-        SqlObject("program", Join(ObsAttachmentTable.ProgramId, ProgramTable.Id))
+        SqlObject("program", Join(ObsAttachmentTable.ProgramId, ProgramTable.Id)),
+        SqlObject("observations")
       )
     )
+
+  lazy val ObsAttachmentElaborator: Map[TypeRef, PartialFunction[Select, Result[Query]]] = 
+    Map(
+      ObsAttachmentType -> {
+        case Select("observations", List(
```

Comment:
This was taken from `ProgramMapping`. I can't see why it works there and has a problem here.